src/codemodder/context.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         self.path_exclude = path_exclude
         self.max_workers = max_workers
         self.tool_result_files_map = tool_result_files_map or {}
+        self.fixing_tool_results = bool(self.tool_result_files_map)
```

Comment:
this is the only clever way I figured to distinguish between a run for find-n-fix codemods versus tool fixing codemods

Reply:
I don't think this will be necessary if we handle this within `FileContext`.